### PR TITLE
Add CeloDance Wallet to Celo use-contractkit

### PR DESCRIPTION
**Pull request by 1035569003**
as a celo mobile wallet, celodance support the transfer and staking of celo assets. after talking with clabs and moola, we propose to integrate celodance wallet to moola. the development work is all finished, so please merge the code, thanks!

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -72,6 +72,34 @@
     "@babel/helper-validator-identifier" "^7.14.9"
     to-fast-properties "^2.0.0"
 
+"@celo-tools/use-contractkit@1.1.0":
```
Comment (jmrossy) [inline yarn.lock:75]: Why has the yarn.lock file changed?

File: packages/use-contractkit/src/constants.tsx [hunk 2]
```diff
@@ -53,6 +55,16 @@ export const PROVIDERS: {
     showInList: () => true,
     listPriority: () => 0,
   },
+  [SupportedProviders.CeloDance]: {
+    name: 'CeloDance',
+    description:
+      'Send、Vote and Earn rewards within one wallet',
```
Comment (jmrossy) [inline packages/use-contractkit/src/constants.tsx:61]: I think you've used the wrong character here for the `,` after 'Send'.
Also please change the casing to 'Send, vote, and earn'

File: packages/use-contractkit/src/constants.tsx [hunk 3]
```diff
@@ -53,6 +55,16 @@ export const PROVIDERS: {
...
+    icon: CELODANCE,
+    canConnect: () => true,
+    showInList: () => true,
+    listPriority: () => 0,
```
Comment (jmrossy) [inline packages/use-contractkit/src/constants.tsx:65]: Please set this to 1

File: packages/use-contractkit/src/walletIcons.tsx [hunk 4]
```diff
@@ -481,3 +481,21 @@ export const CHROME_EXTENSION_STORE: React.FC<React.SVGProps<SVGSVGElement>> = (
     />
   </svg>
 );
+
+export const CELODANCE: React.FC<React.SVGProps<SVGSVGElement>> = (
+  props
+) => (
+  <svg width="981px" height="981px" viewBox="0 0 981 981"   fill="#ffffff" style={{ height: '42px', width: '42px' }} version="1.1" xmlns="http://www.w3.org/2000/svg">
+
```
Comment (jmrossy) [inline packages/use-contractkit/src/walletIcons.tsx:489]: please clean up the spacing

File: packages/use-contractkit/src/walletIcons.tsx [hunk 5]
```diff
@@ -481,3 +481,21 @@ export const CHROME_EXTENSION_STORE: React.FC<React.SVGProps<SVGSVGElement>> = (
     />
   </svg>
 );
+
+export const CELODANCE: React.FC<React.SVGProps<SVGSVGElement>> = (
```
Comment (jmrossy) [inline packages/use-contractkit/src/walletIcons.tsx:485]: rename to CELO_DANCE

File: packages/use-contractkit/src/connectors/connectors.ts [hunk 6]
```diff
@@ -356,3 +356,85 @@ export class WalletConnectConnector implements Connector {
     return wallet.close();
   }
 }
+
+export class CeloDanceConnector implements Connector {
```
Comment (jmrossy) [inline packages/use-contractkit/src/connectors/connectors.ts:360]: Why have you created a new connector? It seems to be just a copy of another connector

**Review by jmrossy — CHANGES_REQUESTED** [review]
Thanks for the PR! Please see the comments below

**Comment by jmrossy** [comment]
@1035569003 In addition to the changes I requested, we'll need to chat with you briefly on Discord before we accept code changes. Your Github account is brand new and has no name. It simply looks too suspicious.

**Comment by bi23com** [comment]
> @1035569003 In addition to the changes I requested, we'll need to chat with you briefly on Discord before we accept code changes. Your Github account is brand new and has no name. It simply looks too suspicious.

hi，it's our new developer and he use his new account. sure please contact us in discord, the handle is  Emma | Bi23 Labs#9980.  thank you very much!

**Pull request closed without merging**